Assets/HoloToolkit/SpectatorView/Scripts/SpatialSync/MarkerDetectionHololens.cs

```diff
@@ -0,0 +1,237 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.using UnityEngine;
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using UnityEngine;
+
+namespace HoloToolkit.Unity.SpectatorView
+{
+    /// <summary>
+    /// Manages the capture process on the HoloLens
+    /// </summary>
+	public class MarkerDetectionHololens : MonoBehaviour
```

Comment:
Not sure if the name really applies here. There's nothing preventing us from detecting markers on any UWP device in this class.